src/Parser/RestoDruid/Modules/Traits/BlessingOfTheWorldTree.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+import Module from 'Parser/Core/Module';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import calculateEffectiveHealing from 'Parser/Core/calculateEffectiveHealing';
+
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+import { formatPercentage } from 'common/format';
+
+const BLESSING_OF_THE_WORLD_TREE_HEALING_INCREASE = 0.03;
+
+/**
+ * Blessing of the World Tree (Artifact Trait)
+ * Increases the healing done by Healing Touch and initial healing done by Regrowth by 3%
+ */
+class BlessingOfTheWorldTree extends Module {
+  static dependencies = {
+    combatants: Combatants,
+  };
+
+  rank = 0;
+  healing = 0;
+
+  on_initialized() {
+    this.rank = this.combatants.selected.traitsBySpellId[SPELLS.BLESSING_OF_THE_WORLD_TREE.id];
+    this.active = this.rank > 0;
+  }
+
+  on_byPlayer_heal(event) {
+    if(event.ability.guid !== SPELLS.HEALING_TOUCH.id || (event.ability.guid !== SPELLS.REGROWTH.id && event.tick) ) {
```

Comment:
Should be &&, not ||. As currently written this can never evaluate true, which is why in practice it's always showing value of 0.

Reply:
Oh, and the && should be an ||. Should be:
```
if(event.ability.guid !== SPELLS.HEALING_TOUCH.id && (event.ability.guid !== SPELLS.REGROWTH.id || event.tick) ) {
```

Reply:
Ooops, done.